winui/Avatar-view/how-to.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about using of the different states that exist in WinUi AvatarView control(SfAvatarView).
```

Comment:
@MadhumithanJayapathi Learn about how to set the different states of badge for the WinUI AvatarView control.

Reply:
@Manivannan-E Changed the content as mentioned.